gtsam/linear/GaussianBayesNet.h

```diff
@@ -177,6 +177,13 @@ namespace gtsam {
      */
     VectorValues backSubstituteTranspose(const VectorValues& gx) const;
 
+    /// print graph
+    virtual void print(
```

Comment:
no virtual. I recommend checking the warnings in a clang CI run to be sure I did not miss anything.

Reply:
Great idea. And yes I'll get rid of the virtuals where there are overrides.